server/helpers/utils.ts

```diff
@@ -1,3 +1,5 @@
+import { promises as fsp } from 'fs'
```

Comment:
don't use that pattern, and rather directly use promisable functions from `fs-extra`